packages/utils/src/document/document-query-progress/handle-conversion-load.ts

```diff
@@ -0,0 +1,93 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+
+import { DocumentQuery, MetriportMedicalApi } from "@metriport/api-sdk";
+import { MedicalDataSource } from "@metriport/core/external/index";
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import duration from "dayjs/plugin/duration";
+import dayjs from "dayjs";
+import axios from "axios";
+dayjs.extend(duration);
+
+const apiKey = getEnvVarOrFail("API_KEY");
+const apiUrl = getEnvVarOrFail("API_URL");
+const cxId = getEnvVarOrFail("CX_ID");
+
+export const internalApi = axios.create({
+  baseURL: apiUrl,
+  headers: { "x-api-key": apiKey },
+});
+
+const metriportApi = new MetriportMedicalApi(apiKey, {
+  baseAddress: apiUrl,
+});
+
+const patientId = "018b67ca-f9f5-7baf-badc-63a18e1a7035";
+const NUM_OF_REQUESTS = 1000;
+
+const docQueryProgress: DocumentQuery = {
+  download: {
+    status: "completed",
+    total: 1,
+    successful: 1,
+    errors: 0,
+  },
+  convert: {
+    status: "processing",
+    total: NUM_OF_REQUESTS,
+    successful: 0,
+    errors: 0,
+  },
+};
+
+/**
+ * Utility to test the load for the conversion status endpoint.
+ *
+ * This will:
+ *   - we will set the doc query progress for the overall and specified hie
+ *   - we will then set the conversion status for given number of requests
+ *   - we will then check the doc query progress for the patient is correct
+ *
+ * Update the respective env variables and run `ts-node handle-conversion-load.ts`
+ *
+ */
+
+async function main() {
+  try {
+    await internalApi.post("/internal/docs/override-progress", docQueryProgress, {
+      params: {
+        cxId,
+        patientId,
+        hie: MedicalDataSource.COMMONWELL,
+      },
+    });
+
+    for (let i = 0; i < NUM_OF_REQUESTS; i++) {
+      await internalApi.post("/internal/docs/conversion-status", null, {
+        params: {
+          patientId,
+          cxId,
+          status: "success",
+          source: MedicalDataSource.COMMONWELL,
+          jobId: `jobId-${i}`,
+        },
+      });
+
+      const queryStatus = await metriportApi.getDocumentQueryStatus(patientId);
+
+      console.log("queryStatus", i, JSON.stringify(queryStatus.convert, null, 2));
+    }
+  } catch (error) {
+    console.error("Error", error);
+  }
+
+  const queryStatus = await metriportApi.getDocumentQueryStatus(patientId);
+
+  console.log("queryStatus", JSON.stringify(queryStatus, null, 2));
+
+  const isComplete = queryStatus.convert?.status === "completed";
+
+  console.log("isComplete", isComplete);
```

Comment:
Thats what i was trying to convery with isCompleted but ill make it more clear. Were trying to test to see if any got missed because of the race condition which would make the status be processing.  Ill make more clear